### PR TITLE
Feature set same site for cookies

### PR DESCRIPTION
**Pull request by julian-berbel**
We got an email from Auth0 warning us about upcoming changes to cookies' behavior in browsers. These changes would cause our iframes to break.

Starting soonish, cookies will _not_ be sent to server when SameSite is None and cookie is not secure.

So, in order for our cookies to work we need:
- in production: Secure cookies (we already have this) and SameSite None (this last thing is because other values would break iframes embedded in sites not our own)
- in development: Insecure cookies (not really a requirement, but then we'd have to setup development over https which is a huge pain) and SameSite Lax (that way cookies will be sent to dev server)

### CODE REVIEW

File: lib/mumukit/login/helpers/login_controller_helpers.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module Mumukit::Login::LoginControllerHelpers
 
   def login_current_user!
-    mumukit_controller.write_cookie!(:login_organization, organization_name)
+    mumukit_controller.mucookie.write!(:login_organization, organization_name)
```
Comment (flbulgarelli) [inline lib/mumukit/login/helpers/login_controller_helpers.rb]: Is this change necessary?

Comment (julian-berbel) [reply lib/mumukit/login/helpers/login_controller_helpers.rb]: Well... as I said the other day, our login seems to be broken in iframes because it fails to access sessionStorage so I guess it wouldn't make a difference either way ¯\\\_(ツ)_/¯.

That being said without this change this cookie would be set with no SameSite attribute, which would make it default to Lax and not be sent in iframes which _could_ break login if it had been working in the first place.
